app/cdap/components/Connections/Browser/SidePanel/CategorizedConnections.tsx

```diff
@@ -156,6 +183,105 @@ export function CategorizedConnections({
     }
   };
 
+  function getConnectionConfig(conn) {
```

Comment:
Nit: I would personally prefer for functions that don't depend on hooks to be at the top level, rather than nested.